drivers/disk/Kconfig.sdmmc

```diff
@@ -61,6 +61,21 @@ config SDMMC_STM32
 	help
 	  File system on sdmmc accessed through stm32 sdmmc.
 
+config SDMMC_STM32_INTERNAL_DMA
+	bool "STM32 SDMMC Internal DMA"
+	depends on SDMMC_STM32
+	depends on SOC_STM32L4Q5XX ||  \
+			SOC_STM32L4P5XX || \
+			SOC_STM32L4R5XX || \
+			SOC_STM32L4R7XX || \
+			SOC_STM32L4R9XX || \
+			SOC_STM32L4S5XX || \
+			SOC_STM32L4S7XX || \
+			SOC_STM32L4S9XX
```

Comment:
We should use a dts property to describe that a idma is available.
In future, this could be extended to L5/h7/u5...